lib/ansible/modules/packaging/language/pip.py

```diff
@@ -472,30 +472,35 @@ def setup_virtualenv(module, env, chdir, out, err):
         module.exit_json(changed=True)
 
     cmd = module.params['virtualenv_command']
+    cmd_args = ""
+
+    if cmd.count(" ") > 0:
+        cmd, cmd_args = module.params['virtualenv_command'].split(" ", 1)
```

Comment:
What happens if the path to virtualenv has a space in it?  You probably need shlex.split instead

Reply:
Good call, I'll revise.